### ISSUE TITLE
Improve SIP history

### ISSUE THREAD

**Issue by martijnhoekstra** [scala/docs.scala-lang#1033]
It can be very tricky finding out what exactly happened to a SIP over its life-cycle. I would like to propose an improvement to the SIP template, where for each SIP meeting the SIP where the SIP was put on the agenda, there is a brief note on what was discussed, with links to the video and to the minutes.

I think this adds value to the history section already present, since they track slightly different things. One tracks the change made to the SIP, while the other tracks decisions made about the SIP.

For example, I could imagine it looking something like this for SIP 35

SIP meeting | minutes | decision | vote
--------------|----------|----------|----
Sep 21, 2017|https://docs.scala-lang.org/sips/minutes/2017-09-21-sip-minutes.html|Numbered|9/9
Oct 24, 2017|https://docs.scala-lang.org/sips/minutes/2017-10-24-sip-minutes.html|Tabled|acclamation
Dec 6, 2017|(https://docs.scala-lang.org/sips/minutes/2017-12-06-sip-minutes.html)|Tabled|N/A
Mar 8, 2018|Not yet published|iterate|acclamation

It would allow for someone interested in some SIP to quickly find out what happened to it, when, and look up the minutes. This would make answering questions like "When was this SIP numbered" a lot easier to answer.

**Comment by jvican** [comment]
I agree this is desirable, will try to find some time before our next SIP meeting to submit a PR adding this rule and modifying the current template. Thanks @martijnhoekstra :smile:.
